WcaOnRails/config/environments/production.rb

```diff
@@ -2,6 +2,13 @@
 
 require "active_support/core_ext/integer/time"
 
+root_url = URI.parse(EnvVars.ROOT_URL)
+Rails.application.routes.default_url_options = {
+  protocol: root_url.scheme,
+  host: root_url.host,
+  port: root_url.port,
+}
+
```

Comment:
We should mirror it for `environments/test.rb` as well :D

Reply:
yep!